specification/customerlockbox/resource-manager/Microsoft.CustomerLockbox/preview/2018-02-28-preview/customerlockbox.json

```diff
@@ -0,0 +1,359 @@
+    "Display": {
+      "type": "object",
+      "properties": {
+        "provider": {
+          "type": "string"
+        },
+        "resource": {
+          "type": "string"
+        },
+        "operation": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        }
+      }
+    },
+    "OperationResult": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Gets or sets action name",
+          "type": "string"
+        },
+        "isDataAction": {
+          "description": "Gets or sets a value indicating whether it is a data plane action",
+          "type": "string"
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay",
+          "description": "Gets or sets Display properties"
+        },
+        "properties": {
+          "description": "Gets or sets properties",
+          "type": "string"
+        },
+        "origin": {
+          "description": "Gets or sets origin",
+          "type": "string"
+        }
+      }
+    },
+    "OperationDisplay": {
+      "description": "Display class",
+      "type": "object",
+      "properties": {
+        "provider": {
+          "description": "Gets or sets RP friendly name",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Gets or sets Resource type friendly name",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Gets or sets operation name",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or sets operation description",
+          "type": "string"
+        }
+      }
+    },
+    "LockboxRequestResponse": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LockboxRequestResponseProperties"
+        }
+      }
+    },
+    "LockboxRequestResponseProperties": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "justification": {
+          "type": "string"
+        },
+        "status": {
+          "enum": [
+            "Initializing",
+            "Pending",
+            "Completing",
+            "Completed",
+            "Unknown"
+          ],
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "expirationDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "duration": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "resourceId": {
+          "type": "string"
+        },
+        "resourceType": {
+          "type": "string"
+        },
+        "supportRequest": {
```

Comment:
what does this indicate?

Reply:
This indicates the Support Request associated with this Lockbox Request. Lockbox is primarily used for the Support Case scenario and so a LB request should only be made if it has a corresponding Support Request made by the customer.